flow/project.py

```diff
@@ -685,24 +698,14 @@ class FlowGroupEntry:
         The :meth:`FlowProject.run` options to pass when submitting the group.
         These will be included in all submissions. Submissions use run
         commands to execute.
-
+    aggregator : :class:`~.aggregator`
+        aggregator object associated with the :class:`FlowGroup`.
```

Comment:
This is not quite true, `None` doesn't necessarily mean that the default value is used. When used for a `FlowGroup` (created using `FlowProject.make_group`), only then `None` would mean the default aggregation is used. See this comment for reference

```
# We register aggregators associated with operation functions
# in `_register_groups` and hence the aggregator parameter is passed as None.
# This is because we do not restrict the decorator placement in terms of
# `FlowGroupEntry`, `aggregator`, or `operation`.
```

Reply:
@b-butler Can you take a look at this? `FlowGroupEntry` allows `group_aggregator=None` but `make_group` replaces `group_aggregator=None` with `group_aggregator=aggregator.groupsof(1)`.